README.md

```diff
@@ -17,14 +17,16 @@ The following steps should only be used to install if you are planning on contri
 1. Install [Node.js 16.x](https://nodejs.org/en/download/)
 1. Install [Rust](https://www.rust-lang.org/learn/get-started).
 1. Install [Yarn](https://classic.yarnpkg.com/en/docs/install).
-1. Windows:
+1. For Windows only:
    1. Install [Build Tools for Visual Studio 2019](https://docs.microsoft.com/en-us/visualstudio/releases/2019/history#release-dates-and-build-numbers), including the C++ Build Tools and the Windows 10 SDK, for Rust.
    1. Rust builds also require installing [MinGW-w64 via win-builds](http://win-builds.org/doku.php/download_and_installation_from_windows).
       1. Choose `Native Windows`, `x86_64`, choose an empty directory, and click OK.
       1. On the next screen, click `Process`.
       1. Once it finishes, add the `bin` folder containing `cc` to your PATH environment variable.
-1. Run `yarn install` from the root directory to install packages.
+1. Run `yarn install` from the root directory to install packages (see sections below for OS specific details).
 
+### Troubleshooting
+1. MacOS:
```

Comment:
These aren't macOS-specific issues, (you'll encounter missing cmake and openssl on Windows as well), so I don't think we should add OS headers here to the troubleshooting. Otherwise looks good!